packages/shared/src/graphql/sources.ts

```diff
@@ -85,3 +92,45 @@ export const SOURCE_QUERY = gql`
     }
   }
 `;
+
+export const SOURCES_QUERY = gql`
+  query Sources($filterOpenSquads: Boolean) {
+    sources(filterOpenSquads: $filterOpenSquads) {
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+      edges {
+        node {
+          ...SourceBaseInfo
+          headerImage
+          color
```

Comment:
@sshanzel I suggested this as it's only applicable to the squad directory

Reply:
I agree, this is not common for every source request. Let's keep this here only.